docs/data-sources/user.md

```diff
@@ -20,7 +29,7 @@ The following arguments are supported:
 
 * `user_id` - (Optional) The ID of the user.
 
-**Note**: only one of email, user_id or username must be provided.
+> **Note**: only one of email, user_id or username must be provided.
```

Comment:
```suggestion
-> Only one of email, user_id, or username must be provided.
```

Reply:
Hey @mattkasa, thanks for the review. I have applied all your suggestions.